### DISCUSSION TITLE
in blockscout manual deployment ,  my

### DISCUSSION THREAD

**Discussion by Anmol-magician1** [blockscout/blockscout#10883] [Support]
-
![Screenshot 2024-10-07 at 11 49 04 AM](https://github.com/user-attachments/assets/c7ce7fb0-0a64-42dc-a685-26b1e4644cab)

my stast are serving but getting resposne error

**Comment by GimluCom** [comment]
Please try running a nginx proxy with this config https://github.com/blockscout/blockscout/blob/v6.8.1-beta/docker-compose/proxy/default.conf.template#L1-L34
